codeexamples/SYCLUsingCUDAStaticLibraryOfKernels/StaticLibraryBuildCUDAKernels/.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,18 @@
+{
+    "configurations": [
+        {
+            "name": "Linux",
+            "includePath": [
+                "${workspaceFolder}/**",
+                "${workspaceFolder}/../../cuda-samples-nvidia/Common",
+                "/usr/local/cuda-12.0/targets/x86_64-linux/include"
+            ],
+            "defines": [],
+            "compilerPath": "/usr/local/cuda/bin/nvcc",
```

Comment:
Similarly this isn't guaranteed to be the location of `nvcc`. Does `compilerPath` respect the `PATH` environment variable?

Reply:
Same answer to the previous comment.

Reply:
So this file is the best it can be and being included does allow a user to edit the path to fit their needs by referencing an example. Information has been given in the readme for this project on the purpose of the (set of four) configuration files linking to a blog on how to set up VSCode for DPC++.